src/commands/superGenezio/superGenezio.ts

```diff
@@ -48,11 +45,6 @@ export async function genezioCommand() {
                     installDeps: true,
                 };
 
-                if (!(await getAuthToken())) {
```

Comment:
why did you remove this?

Reply:
Because the auto login is done now in the `deployCommand` function. We no longer need to auto login here.

Reply:
let s also keep it here